DependencyInjection/ContentfulExtension.php

```diff
@@ -72,8 +73,12 @@ protected function loadDeliveryClient($name, array $client, ContainerBuilder $co
             $options['cacheDir'] = '%kernel.cache_dir%' . '/contentful';
         }
 
+        $deliverClientDefinition = class_exists(ChildDefinition::class) ?
```

Comment:
I find it amazing that this actually works, and does not trigger an error/exception.

PHP 🤷‍♂️ 😁